src/components/header/index.stories.mdx

```diff
@@ -0,0 +1,98 @@
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+import { Box } from '@chakra-ui/react';
```

Comment:
Can you use our component `Box` instead importing it through '@chakra-ui/react'? Thanks